source/_integrations/filter.markdown

```diff
@@ -65,7 +66,7 @@ name:
   required: false
   type: string
 unique_id:
-  description: An ID that uniquely identifies the filter sensor. Set this to a unique value to allow customization through the UI.
+  description: An ID that uniquely identifies the filter sensor. Set this to a unique value to allow customization through the UI. If you do not define it, it will be automatically derived from the ´name´ by HA
```

Comment:
This is not correct, the unique ID will not be derived from the name.

Reply:
Unsatisfying answer. There probably is an internal unique Identifier. How is that created? (Do not say "No", but rather give the/a correct explanation)

Reply:
It isn't created 🤷 
This line added is simply not true.

Reply:
OK. Killer argument. I withdraw that part of my change :-(